data.tf

```diff
@@ -0,0 +1,18 @@
+data "aws_ami" "amazonlinux" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["amzn2-ami-kernel-*"]
+    }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+  owners = ["137112412989"]
+}
+
+data "aws_availability_zones" "available" {
+  state = "available"
+}
```

Comment:
Add newline at the end of file.

Reply:
Its my bad , now I have added the new line .